src/decaylanguage/dec/dec.py

```diff
@@ -655,7 +660,7 @@ def list_decay_modes(self, mother, pdg_name=False):
             for mode in self._find_decay_modes(mother)
         ]
 
-    def _decay_mode_details(self, decay_mode):
+    def _decay_mode_details(self, decay_mode, keep_photos=False):
```

Comment:
`display_photos_keyword` might be a better name. And now it makes sense to add a proper docstring to the function as pretty much elsewhere.

Reply:
Done